src/librustc/lint/levels.rs

```diff
@@ -470,6 +472,32 @@ impl<'a> LintLevelsBuilder<'a> {
         }
     }
 
+    /// If we have e.g. `#[allow(future_compat_lint)]` this will have
```

Comment:
My initial reading of this comment is that its referencing a (non-existant) lint named `future_compat_lint`, as if we had some way to allow then en masse, rather that it being a placeholder.

Maybe phrase instead as: "If we have e.g. `#[allow(some_future_compat_lint)]` ...", to make it clearer that this is not a concrete thing you are naming here?

Reply:
Maybe even `#[allow($some_future_compat_lint)]` to clarify that it's a placeholder?